Backpack/src/androidTest/java/net/skyscanner/backpack/util/ThemesUtilTest.kt

```diff
@@ -16,34 +18,98 @@ import org.junit.runner.RunWith
 
 @RunWith(AndroidJUnit4::class)
 class ThemesUtilTest {
-  private lateinit var context: Context
   private lateinit var activity: AppCompatActivity
 
   @get:Rule
   internal var activityRule: ActivityTestRule<TestActivity> =
     ActivityTestRule(TestActivity::class.java)
 
+  private val blue500 by lazy { ContextCompat.getColor(activity, R.color.bpkBlue500) }
+  private val gray50 by lazy { ContextCompat.getColor(activity, R.color.bpkGray50) }
+  private val gray100 by lazy { ContextCompat.getColor(activity, R.color.bpkGray100) }
+  private val gray300 by lazy { ContextCompat.getColor(activity, R.color.bpkGray300) }
+  private val gray500 by lazy { ContextCompat.getColor(activity, R.color.bpkGray500) }
+  private val gray700 by lazy { ContextCompat.getColor(activity, R.color.bpkGray700) }
+  private val gray900 by lazy { ContextCompat.getColor(activity, R.color.bpkGray900) }
+
+  private val customBlue = Color.parseColor("#9B104A")
+  private val customGrey50 = Color.parseColor("#9B104B")
+  private val customGrey100 = Color.parseColor("#9B104C")
+  private val customGrey300 = Color.parseColor("#9B104D")
+  private val customGrey500 = Color.parseColor("#9B104E")
+  private val customGrey700 = Color.parseColor("#9B104F")
+  private val customGrey900 = Color.parseColor("#9B105A")
```

Comment:
We should standardise on the spelling here - these lines use 'grey' but the earlier ones use 'gray'. (gray is American spelling, grey is British). We use the US spelling everywhere else so let's go with 'gray'.

Reply:
Done